addons/weapononback/script_component.hpp

```diff
@@ -0,0 +1,46 @@
+#define COMPONENT weapononback
+#define COMPONENT_BEAUTIFIED Weapon on Back
+#include "\z\ace\addons\main\script_mod.hpp"
+
+// #define DEBUG_MODE_FULL
+#define DISABLE_COMPILE_CACHE
+// #define ENABLE_PERFORMANCE_COUNTERS
+
+#ifdef DEBUG_ENABLED_WEAPONONBACK
+    #define DEBUG_MODE_FULL
+#endif
+
+#ifdef DEBUG_SETTINGS_WEAPONONBACK
+    #define DEBUG_SETTINGS DEBUG_SETTINGS_WEAPONONBACK
+#endif
+
+#include "\z\ace\addons\main\script_macros.hpp"
+
+// Inventory IDD
```

Comment:
self-describing name
```suggestion
```

Reply:
I'd rather have too many comments than too few. I agree that it's unnecessary on a few, for example this one, but I added them to all of them for consistency.

Reply:
Those comments split defines, keep them.